### ISSUE TITLE
Script contains SQL Vulerabilities and Backdoors!

### ISSUE THREAD

**Issue by satx42** [felinegambler/CryptoDice#1]
This script contains poorly written code and is completely (likely intentionally) vulnerable to SQL injection attacks. The attackers can make unlimited withdrawals and steal all of your money.

Contact me at  scripts@dice.gq  for the repaired version. It is completely safe and secure. :)

**Comment by kizivienne** [comment]
Yes be VERY CAREFUL this seller was reported a proved SCAMMER on bitcointalk.org, there is in the code this backdoor the seller can log into your site anytime, like an admin and run with your coins !
